docs/data/_index.md

```diff
@@ -57,28 +59,28 @@ Experience streamlined data handling with advanced querying, viewing, and filter
 {{< cards >}}
 {{% manualcard link="/data/query/" %}}
 
-#### Data querying
+#### Query data
 
 Make SQL or MQL queries on synced sensor data, accessible through the Viam app and MQL-compatible clients.
 
 {{% /manualcard %}}
 {{% manualcard link="/data/view/" %}}
 
-#### Viewing and filtering data
+#### View and filter data
 
 View and filter different data types in the cloud, with the option to delete data on the Viam app.
 
 {{% /manualcard %}}
 {{% manualcard link="/data/dataset/" %}}
 
-#### Creating datasets
+#### Create datasets
 
 Label data for management and machine learning, with dynamic datasets that change with underlying data modifications.
 
 {{% /manualcard %}}
 {{% manualcard link="/data/export/" %}}
 
-#### Exporting data
+#### Export data
```

Comment:
Ah yes, I agree with these changes: gerund --> simple imperative. Think many of these were me, and this is better!